wiki/Tournaments/BQT/6/en.md

```diff
@@ -0,0 +1,388 @@
+5. If a team CANNOT meet at any of the scheduled times, please DM/Ping @Admin to arrange a custom lobby. Please note that the custom lobbies are not guaranteed as they are based on staff availability.
+6. If a player disconnects during the first 30 seconds of the map, that specific player must stay at the end of the lobby and replay that map by themselves.
+
+### Match instructions
+
+1. Matches are made approximately 5-10 MINUTES before the matches are scheduled to start.
+2. All match lobbies will be named `(Team 1) vs (Team 2)`, with players from Team 1 playing in slots #1 and #2 (Red Team) and players from Team 2 playing in slots #3 and #4 (Blue Team).
+3. If there are less than 2 players from either team WITHIN THE LOBBY, there is a 15 minute grace period provided to allow them to join; **10 minutes - team lose ban**, **15 minute - FF**. The team with the most players within the lobby by the 15 minute mark will win by default. If both teams have only one player present on each side and both of their 2nd teammate doesn't show up by the 15 minute grace period, it'll be FF on both sides.
+4. There will be **NO WARMUPS** this season, so please warmup before your match.
+5. Each team will use `!roll` for the banning and picking phase. The winner of the roll will get to **choose pick order/ban order** The loser of the roll will get to choose between the unchosen options. For Example... If Team 1 rolls 100 and Team 2 rolls 50... Team 1 wants to choose ban order & they may choose ban first or second...Then that leaves Team 2 to choose from pick order. They may choose to pick first or second.
+6. There will be 1 BAN PER TEAM on matches up to RO16, and 2 BANS PER TEAM (ABBA format) on matches from Quarterfinals onwards. **Each team may not ban two maps from the same mod pool in a match**, except for NoMod maps.
+7. Each team will be allowed 60 seconds to ban their map. When 2 bans are enabled, teams are **NOT allowed to ban from the same mod** (NM is the exception). For example, Team 1 can't ban both HD1 and HD2 during their banning phase.
+8. Picks will happen in alternating order after the previous map is finished. Each team has 120 seconds to pick their map. If a map is not picked by 120 seconds, the pick will go to the other team.
+9. **Picking maps from the same mod pool consecutively in a match is NOT ALLOWED**, except for NoMod maps. For example, when a team picks HR1, they can't pick HR2 in their next turn. They had to pick something else first (e.g. DT1) before they can pick HR2.
+10. A map will be aborted if a player disconnects within the first 30 seconds of the map. If there is a disconnect and the map is not aborted, the score for that player will be counted as 0 UNLESS a score is provided. AN ABORT IS ALLOWED ONCE PER TEAM, PER MATCH.
+11. NoFail will be enforced on every map.
+12. Freemod rules : Each players in a team must have a unique mod! One player with HD/EZ/HDEZ and another player with HR/HDHR . EZ = 1.8x (with potentially HD)
```

Comment:
```suggestion
12. When playing FreeMod, each player in a team must have a unique mod! One player with HD/EZ/HDEZ and another player with HR/HDHR. EZ = 1.8x (with potentially HD)
```

wording still feels lackluster so i'd appreciate another maintainer's take on how to adjust this.